app/src/store/data.ts

```diff
@@ -46,22 +49,27 @@ export default function useDataStore() {
     // Get blockdata
     lastBlockNumber.value = BigNumber.from(await provider.value.getBlockNumber()).toNumber();
 
+    // Get computed on grantRoundManager
+    const grantRoundManager = computed(() => _grantRoundManager.value);
```

Comment:
Can we just use the imported `grantRoundManager` from the wallet store directly? It's returned as a computed property to begin with, so it seems here we just wrap it in another computed property

Reply:
Yes 🙌 😄